index.js

```diff
@@ -0,0 +1,187 @@
+// Create the getFullNameFromUser function
+
+const user = {
+    firstName: 'John',
+    lastName: 'Smith'
+}
+function getFullNameFromUser(user) {
+    return user.firstName + ' ' + user.lastName;
+}
+
+console.log(getFullNameFromUser(user));
+
+// Write the createUser function
+
+user.firstName = 'Katie',
+user.lastName = 'Williams'
+
+function createUser(firstName, lastName) {
+    return {
+        fullName: firstName + ' ' + lastName,
+        firstName,
+        lastName
+    }
+}
+
+const userName = createUser(user.firstName, user.lastName);
+console.log(userName);
+
+// write the checkIfPropertyIsDefined function
+const userObject = {
+    height: 170
+}
+function checkIfPropertyIsDefined(userObject, propertyName) {
+    return userObject[propertyName] === undefined;
+}
+
+const checkProperty = checkIfPropertyIsDefined(userObject, 'firstName'); // true
+const checkPropertySecond = checkIfPropertyIsDefined(userObject, 'height'); // false
+console.log(checkProperty);
+console.log(checkPropertySecond);
+
+// Add up all of the salaries
+
+const salaries = {
+    john: 50,
+    adam: 100,
+    katie: 75
+}
+
+const salarySum = salaries.john + salaries.adam + salaries.katie;
+console.log(salarySum);
+
+// Write the isObject function. Use the typeof operator.
+
+function isObject(data) {
+    return typeof data === "object" && data !== null;
+}
+
+console.log(isObject({})); // true
+console.log(isObject(null)); // false
+
+// Write the removeProperty function. It should return true if there is a property with a given key to
+// remove. Otherwise, it should return false.
+
+const userData = {
+    name: 'John',
+}
+
+function removeProperty(userData, propertyData) {
+    return userData.hasOwnProperty(propertyData);
```

Comment:
Your `removeProperty` function does not remove a property.
Also, don't use the `hasOwnProperty` function yet, we will cover it in detail later. Simply check if a property is not `undefined.`